gruvbox-tpm.tmux

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env bash
-
-source "./src/gruvbox-main.sh"
-
-# vim: ai et ft=bash
+#!/bin/bash
+CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
It can work simplified like this

CURRENT_DIR="$(dirname "$(realpath "$0")")", 

but yeah this fix works.

Reply:
Thanks to @jgimeno for PR with the fix! :) 

Yeah, there are tons of ways how to get the current path but not all work in all environments. As could noticed this very well when I broke the main with [another way](https://github.com/egel/tmux-gruvbox/commit/bf07efd3227d01d5b2a97cf04ccd4f5f69a0ddd7) 😜 🙈 (btw I tested this on macOS, and it worked, thou I did not try on Linux - my bad)

Before the next release/v2-0-0, that is btw just behind the corner. I will add a new ticket for testing on Linux & macOS. Hope this will be easy to do and prevent these kinds of errors in the future 🙂